lib/cli/middlewares/logger.js

```diff
@@ -1,22 +1,21 @@
-
+import logger from "@ui5/logger";
+import ConsoleWriter from "@ui5/logger/writers/Console";
 import {getVersion} from "../version.js";
 /**
- * Logger middleware used as one of default middlewares by tooling
+ * Logger middleware used as one of default middleware by tooling
```

Comment:
this sentence sounds quite technical.. 
What about `Logger middleware to enable logging capabilities`?

Reply:
Done